1-Pflichtenheft/pflichtenheft.tex

```diff
@@ -75,6 +75,9 @@ \subsection{Abgrenzungskriterien}
 \section{Produkteinsatz}
 
 \section{Produktumgebung}
+Das Programm soll für das Erstellen und Verwalten der Blockchain das Hyperledger Fabric Framework verwenden, welches eine auf dem Consensus verfahren basierende Blockchain-Implementierung bietet.
+Hierfür werden für die Hauptanwendung das Hyperledger Fabric Java SDK verwendet werden und für  das erstellen von Chaincodes soll Go verwendet werden.
+Um die Funktionalität von Hyperledger Fabric zu gewährleisten werden ebenso eine Installation von node.js, npm sowie der GoLang und Python 2.7 benötigt.
```

Comment:
Maybe parts of this belong to "Spezielle Anforderungen an die Entwicklungsumgebung" ?